scripts/DMF/dmfsnmp.xsd

```diff
@@ -111,6 +111,7 @@
                       <xs:attribute type="xs:string" name="outlet_group" use="optional"/>
                       <xs:attribute type="xs:string" name="writable" use="optional"/>
                       <xs:attribute type="xs:string" name="lookup" use="optional"/>
+                      <!-- Note: setvar was obsoleted and will be ignored by codebase -->
                       <xs:attribute type="xs:string" name="setvar" use="optional"/>
```

Comment:
this should have been removed

Reply:
I think we can do that; this being a schema, it should then actively refuse DMF files that contain a non-conformant (old) field.

I may have been a bit confused by the field still being sort of defined in the C header (as a reserved number to avoid shifting others).